### PR TITLE
Port 5683 k 8 s exporter create default pages

### PR DESCRIPTION
**Pull request by yairsimantov20**

### CODE REVIEW

File: pkg/defaults/defaults.go [hunk 1]
```diff
@@ -108,36 +123,60 @@ func deconstructBlueprintsToCreationSteps(rawBlueprints []port.Blueprint) ([]por
...
 }
 
-func createResources(portClient *cli.PortClient, defaults *Defaults, config *port.Config) *AbortDefaultCreationError {
+func validateResourcesDoesNotExist(portClient *cli.PortClient, defaults *Defaults, config *port.Config) *AbortDefaultCreationError {
+	var errors []error
 	if _, err := integration.GetIntegration(portClient, config.StateKey); err == nil {
-		return &AbortDefaultCreationError{Errors: []error{
-			fmt.Errorf("integration with state key %s already exists", config.StateKey),
-		}}
+
```
Comment (talsabagport) [inline pkg/defaults/defaults.go:147]: if block is empty

File: pkg/defaults/defaults.go [hunk 2]
```diff
@@ -108,36 +123,60 @@ func deconstructBlueprintsToCreationSteps(rawBlueprints []port.Blueprint) ([]por
...
+		}
+	}
+
+	if errors != nil {
+		return &AbortDefaultCreationError{Errors: errors}
+	}
+	return nil
+}
+
+func createResources(portClient *cli.PortClient, defaults *Defaults, config *port.Config) *AbortDefaultCreationError {
```
Comment (talsabagport) [inline pkg/defaults/defaults.go:168]: There is some scale issue with the current code architecture. One or two more resources and this function and the entire file will be too huge to maintain. Up to you if to solve it now

**Review by talsabagport — CHANGES_REQUESTED** [review]
Left two comments 🚀

File: pkg/defaults/defaults.go (hunk 2 above)
Comment (yairsimantov20) [reply pkg/defaults/defaults.go:168]: We plan to handle it in port api so i think its ok for now

**Pull request merged**